### ISSUE TITLE
Uneven grids with Picnic 6

### ISSUE THREAD

**Issue by esoco** [picnicss/picnic#90]
The documentation says that the class "row" will be removed in Picnic 6 and is replaced by the class "grid" as it seems. But the width control classes like "two-third" don't work for child elements of  "grid" in  the current version on github. Is this still work in progress or is there another way to achieve uneven grid widths in Picnic 6?

**Comment by franciscop** [comment]
That is totally true, I mixed the old `.row` with a new and experimental `.grid` in the documentation, but I forgot to update it there. However, I am not sure whether I will be upgrading it to `.grid` or to `.flex`, a new, working small project I made based on flexbox: [**Flexpoint**](http://francisco.io/flexpoint/).

I finished *flexpoint* a couple of days ago and now I'm testing it to check that it works as expected in many situations and that it's really useful. I found both `.row` and `.grid` implementations really limiting in what could be achieved, however I don't want to release a half-baked solution (again).

**Comment by franciscop** [comment]
TL;DR: flexpoint is coming, `.row` is leaving and `.grid` might not be finished at all.

**Comment by esoco** [comment]
Does that mean flexpoint will be a separate project and grids are removed at all from the next Picnic CSS?

**Comment by franciscop** [comment]
No, I will be joining flexpoint into Picnic CSS when it is mature (probably within few days). But if you need it right now you can use it as a separated project. Please report any bug that you might find if you do so. The syntax will probably stay the same and only change the SCSS.

**Comment by esoco** [comment]
Ok, I'll give it a try then. Are there any infos on browser compatibilty?

**Comment by franciscop** [comment]
Same as Picnic CSS, IE11+. It is based on flexbox, so it's not that great for IE: http://caniuse.com/#feat=flexbox

**Comment by franciscop** [comment]
Flexpoint is implemented in Picnic CSS 6.0.0
